### DISCUSSION TITLE
Merging to master

### DISCUSSION THREAD

**Discussion by bcholmes** [bcholmes/StarTrek2d20#139] [General]
Hey folks!

This is a technical topic, and most people probably won't care about it. That's okay.

If you've ever looked in to the code, you'll see that all of the updates to the app that have happened over the last 18 month or more have been sitting on a branch called sta-complete. As I've talked about before (see, for example, #75) I've kept the master branch as a mirror of the original repo to aid in making pull requests back to azjerei's copy of the code. But it's been a long, long time since any PRs have been put back into the original repo, so I think that's probably no longer useful. And it'll be easier for people to find the most up-to-date code if I put it on the master branch.

So that's what I'm going to do. In the next few days, I'll merge the latest code to the master branch, and from that point forward, continue to push my changes there. The sta-complete branch will probably be deleted.

**Comment by Felderburg** [comment]
Why merge, as opposed to just changing the "default" branch in your repo to sta-complete?

**Reply by bcholmes** [reply]
Fair question. I've made that update. I guess we can see how that works out.
